src/test/resources/data/forge/item_tier_ordering.json

```diff
@@ -0,0 +1,3 @@
+{
+  "order": [ "iron" ]
```

Comment:
Doesn't having only one of the various tiers cause issues when reading the file?

Reply:
Err yes I tested the error condition last things before I made the commit. will fix later.